app/views/users/index.html.erb

```diff
@@ -1 +1,13 @@
-<h1>Users Index action</h1>
\ No newline at end of file
+<h1>Users Index action</h1>
+
+<% @users.each do |person| %>
+  <div class="user__post">
+    <%= image_tag "#{person.photo}" %>
+
+  <div class="user__post__info">
+    <h3><%= person.name%></h3>
+    <div class="post__num">
+    <p> Number of posts: <%= person.posts_counter %></p></div>
+  </div>
+  </div>
+<% end %>
```

Comment:
- [Optional] Hi @Strangeal, it seems you are repeating this piece of code since you already defined a partial for this in the view/shared directory.